include/tvm/runtime/packed_func.h

```diff
@@ -554,6 +512,10 @@ class TVMArgValue : public TVMPODValue_ {
       return std::string(value_.v_str);
     }
   }
+  operator tvm::runtime::String() const {
+    // directly use the std::string constructor for now.
+    return tvm::runtime::String(operator std::string());
```

Comment:
@tqchen It happened to me that Line511 above failed for the check because the type_code_ for String is an object. Should we remove this and pass String objectref directly? Or do we need to handle String through FFI?

Reply:
Ah, i see, we will need to add a patch, by checking if the result is kStr and run this, alternatively, use AsObjectRef<String>

Reply:
so let's just `return AsObjectRef<tvm::runtime::String>()` for now?

Reply:
https://github.com/apache/incubator-tvm/pull/5266